src/websockets/libwebsockets/LibWebsocketServer.cpp

```diff
@@ -313,9 +313,9 @@ int LibWebsocketServer::eventCallback(struct lws* wsi, enum lws_callback_reasons
             {
                 // Check URI
 #ifdef _MSC_VER
-                char uri[512u];
+                char* uri = new char[512u];
 #else  // _MSC_VER
-                char uri[lws_hdr_total_length(wsi, WSI_TOKEN_GET_URI) + 1];
+                char* uri = new char[lws_hdr_total_length(wsi, WSI_TOKEN_GET_URI) + 1];
 #endif // _MSC_VER
```

Comment:
If using dynamic allocation, we don't need the `#if _MSVC_VER` guard anymore.
We can keep: `char* uri = new char[lws_hdr_total_length(wsi, WSI_TOKEN_GET_URI) + 1];` for all the compilers.